solr/solr-ref-guide/src/docker-networking.adoc

```diff
@@ -205,7 +205,7 @@ To load data, and see it was split over shards:
 [source,bash]
 ----
 docker exec -it --user=solr zksolr1 bin/post -c my_collection1 example/exampledocs/manufacturers.xml
-# /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -classpath /opt/solr/dist/solr-core-9.0.0.jar -Dauto=yes -Dc=my_collection1 -Ddata=files org.apache.solr.util.SimplePostTool example/exampledocs/manufacturers.xml
+# /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -classpath /opt/solr/server/lib/ext/*:/opt/solr/server/solr-webapp/webapp/WEB-INF/lib/* -Dauto=yes -Dc=my_collection1 -Ddata=files org.apache.solr.util.SimplePostTool example/exampledocs/manufacturers.xml
```

Comment:
if we want to remove duplicate jars, we would still need to remove `dist/solrj-lib`, since those are duplicated in `WEB-INF` and `lib/ext`. But yes, we could likely keep those two in `dist/`.